react-components/select/step6.md

```diff
@@ -0,0 +1,15 @@
+There a couple additional properties used to customize general visual aspects of the Select component.
+
+width controls the overall width of the select toggle, using pixels or percentages.
+height controls the maximum height of the expanded select menu, using pixels or percentages.
```

Comment:
split up this text into 4 actionable points and at each point add the property to the select component.

Reply:
There a couple should be there are additional properties...